stack.yaml

```diff
@@ -6,6 +6,7 @@ extra-deps:
   # --- Missing from Stackage --- #
   - QuickCheck-GenT-0.2.0
   - bloomfilter-2.0.1.0
+  - compactable-0.1.2.3
```

Comment:
since this is just for `fforMaybe`, I say we just vendor that code in an monomorphize it at its call site rather than adding the whole library as a dep

Reply:
With proper compilation with `-split-sections` (or even a post-compile `strip` call) this doesn't become so much of an issue - all the dead code is removed.

Reply:
Agree with Emily -- pull in the definition of the combinator you need vs. making us link with yet another abstruse utilities library